### PR TITLE
Add data_source label

### PR DESCRIPTION
**Pull request by andersonfrailey**
This PR adds `data_source` to the list of possible labels. This will hopefully solve #59.

cc @hdoupe

### CODE REVIEW

**Comment by hdoupe** [comment]
Looks good to me @andersonfrailey. You may want to add a test for a parameter that is only available for one of the data sets. This reform causes a crash:

```json
{
    "policy": {
        "BEN_ssi_repeal": [
            {
                "year": 2019,
                "value": true,
                "data_source": "CPS"
            }
        ]
    },
    "behavior": {}
}
```

You could add the benefit parameter to [this test](https://github.com/PSLmodels/Tax-Brain/blob/master/compconfig/compconfig/tests/test_functions.py#L77-L103) so it wouldn't have any significant effects on the run time of the `compconfig` tests.

**Comment by andersonfrailey** [comment]
Good idea, @hdoupe. Added in the latest commit.

File: compconfig/compconfig/tests/test_functions.py [hunk 1]
```diff
@@ -85,6 +85,9 @@ def test_convert_adj():
             {"year": 2019, "value": -0.001}
         ],
         "EITC_c": [{"EIC": "0kids", "year": 2019, "value": 1000.0}],
+        "BEN_ssi_repeal": [
+            {"year": 2019, "value": True}
```
Comment (hdoupe) [inline compconfig/compconfig/tests/test_functions.py]: This should have a data source key too.

```python
         "BEN_ssi_repeal": [
            {"year": 2019, "data_source": "CPS", "value": True}
```

File: compconfig/compconfig/helpers.py [hunk 2]
```diff
@@ -123,13 +123,18 @@ def convert_adj(adj, start_year):
             continue
         for valobj in valobjs:
             # has keys "year" and "value"
```
Comment (hdoupe) [inline compconfig/compconfig/helpers.py]: @andersonfrailey you can probably delete this comment. The new condition makes it clearer that `valobj` does not have keys outside of value, data_source, and year.

**Comment by hdoupe** [comment]
LGTM!

**Pull request merged**